lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -307,6 +308,73 @@ def detach(self, ue_id, reason_type, wait_for_s1_ctxt_release=True):
         with self._lock:
             del self._ue_ip_map[ue_id]
 
+    def verify_flow_rules(self, flowRules):
```

Comment:
this implementation is still very test case specific. Please see my other comments on what to pass as flow rules to be verified.